### PR TITLE
Include babel handbook

### PR DESCRIPTION
**Pull request by xtuc**
Refers to #1000.

I created a page `/docs/handbook/`.

babel-handbook

@thejameskyle what do you think ?

### CODE REVIEW

**Comment by xtuc** [comment]
We could remove the table of contents. It is redundant with the right sidebar.

```
| slice: 73, 100000
```

Or remove the right sidebar.

**Comment by hzoo** [comment]
We can't really slice it in that way because we will probably add/remove headings in the future

**Comment by xtuc** [comment]
Ok, Well, we will just add more shift for those new header I guess.

**Comment by hzoo** [comment]
Do you think we should do the shift thing and then update it with handbook updates?

**Comment by xtuc** [comment]
I don't know because it will require unnecessary maintenance.

We could remove the right bar to avoid the duplicate table of content.

/cc @existentialism what do you think ?

**Comment by hzoo** [comment]
Probably easier to remove the right bar instead

**Comment by xtuc** [comment]
I removed the right bar.

handbook

**Comment by xtuc** [comment]
We need to link it somewhere, maybe in the core-packages section (http://babeljs.io/docs/core-packages/)

**Comment by hzoo** [comment]
it's also at the bottom of the homepage atm

screen shot 2016-12-14 at 12 33 36 pm

Should link it where we want to talk about making plugins

**Comment by xtuc** [comment]
Isn't it confusing for new user to have the button on the homepage ?

There is a section for that purpose: http://babeljs.io/docs/plugins/#plugin-development

**Comment by hzoo** [comment]
yeah we can remove it too

**Comment by xtuc** [comment]
@hzoo @existentialism Can we merge this?

**Comment by hzoo** [comment]
Yeah just have the merge conflicts

File: index.md [hunk 1]
```diff
@@ -346,13 +346,6 @@ export default function ({types: t}) {
     </div>
   </div>-->
 
-  <div class="text-center">
-    <div class="btn-wrapper btn-wrapper-lg">
-      <a href="{{ site.baseurl }}/docs/learn-es2015/" class="btn btn-featured">Learn ES2015</a>
-      <a href="{{ site.baseurl }}/docs/setup/" class="btn btn-featured">Set up Babel</a>
-    </div>
-  </div>
-
```
Comment (xtuc) [inline index.md]: Since #1108

**Comment by existentialism** [comment]
Really wish we could skip rendering the ToC and use the right bar, but we can improve it later.

Couple things:

- Edit this page link doesn't work
- Some internal anchor links don't work (for instance `#replacing-a-node-with-multiple-nodes` vs `#toc-replacing-a-node-with-multiple-nodes`)
- Should we augment code block titling to support `diff`? Should we disable copy/paste for these blocks? (probably diff PR)

File: docs/handbook.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ layout: page
 title: Babel Plugin Handbook
 description: This document covers how to create Babel plugins.
 permalink: /docs/handbook/
+package_source: babel-handbook
+package: babel-handbook
```
Comment (xtuc) [inline docs/handbook.md]: At least now the Github button works, "Edit this page" redirect to the README of the handbook and the package in npm is not found

**Pull request closed without merging**